src/SFA.DAS.EmployerCommitmentsV2.Web/Startup/DataProtectionStartupExtensions.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Microsoft.AspNetCore.DataProtection;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using SFA.DAS.EmployerCommitmentsV2.Configuration;
+using SFA.DAS.EmployerCommitmentsV2.Web.Configurations;
+using StackExchange.Redis;
+
+namespace SFA.DAS.EmployerCommitmentsV2.Web.Startup
+{
+    public static class DataProtectionStartupExtensions
+    {
+        public static IServiceCollection SaveKeysToStackExchangeRedis(this IServiceCollection services, IConfiguration configuration, IHostingEnvironment environment)
```

Comment:
Maybe this could be called `AddDataProtection`, as that's the first extension method that is called and fundamentally what is happening here.